tools/swift-format/Sources/Rules/BlankLineBetweenMembers.swift

```diff
@@ -17,5 +17,148 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#vertical-whitespace
 public final class BlankLineBetweenMembers: SyntaxFormatRule {
+  public override func visit(_ node: MemberDeclBlockSyntax) -> Syntax {
+    var membersList = [MemberDeclListItemSyntax]()
+    var hasValidNumOfBlankLines = true
+    
+    // Iterates through all the declaration of the member, to ensure that the declarations have
+    // at least on blank line and doesn't exceed the maximum number of blank lines.
+    for member in node.members {
+      let currentMember = checkForNestedMembers(member)
+      guard let memberTrivia = currentMember.leadingTrivia else { continue }
+      let triviaWithoutTrailingSpaces = memberTrivia.withoutTrailingSpaces()
+      guard let firstPiece = triviaWithoutTrailingSpaces.first else { continue }
+      
+      if exceedsMaxBlankLines(triviaWithoutTrailingSpaces) {
+        let correctTrivia = removeExtraBlankLines(triviaWithoutTrailingSpaces, currentMember)
+        let newMember = replaceTrivia(on: currentMember, token: currentMember.firstToken!,
```

Comment:
When a function call (or definition) wraps, use the style described in our style guide:

```swift
let newMember = replaceTrivia(
  on: currentMember,
  token: currentMember.firstToken!
  leadingTrivia: correctTrivia
) as! MemberDeclListItemSyntax
```

(Apply to all wrapped functions in the file.)